pkgs/by-name/et/ethercat/common.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  fetchFromGitLab,
+  stdenv,
+  gitUpdater,
+  autoreconfHook,
+  pkg-config,
+  withDocs ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ethercat";
+  version = "1.6.1";
+
+  src = fetchFromGitLab {
+    owner = "etherlab.org";
+    repo = "ethercat";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Ki8WUKUW4O9SNFTVIXrWpL6suCu7j2nLQtpWVsUoiK0=";
+    fetchSubmodules = withDocs;
+    leaveDotGit = withDocs;
+  };
+
+  meta = {
```

Comment:
meta should be last

Reply:
Fixed in e792a4370cc0140573d19c4013ce7b375f464d10